plugins/modules/openwrt_init.py

```diff
@@ -32,20 +32,19 @@
   state:
     type: str
     description:
-      - V(started)/V(stopped) are idempotent actions that will not run commands unless necessary.
-      - V(restarted) will always bounce the service.
-      - V(reloaded) will always reload.
+      - V(started)/V(stopped) are idempotent actions that do not run commands unless necessary.
+      - V(restarted) always bounces the service.
+      - V(reloaded) always reloads.
     choices: ['started', 'stopped', 'restarted', 'reloaded']
   enabled:
     description:
-      - Whether the service should start on boot. B(At least one of state and enabled are required).
+      - Whether the service should start on boot. B(At least one of O(state) and O(enabled) are required).
```

Comment:
Nested markup doesn't work. How about:
```suggestion
      - Whether the service should start on boot. B(At least one) of O(state) and O(enabled) are required.
```
or
Nested markup doesn't work. How about:
```suggestion
      - Whether the service should start on boot. B(At least one of) O(state) B(and) O(enabled) B(are required.)
```
(Since `O(...)` for an option without value renders it in bold anyway, the second suggestion has the same effect as your original version intended to have.)